frontend/src/components/TradeBox/index.tsx

```diff
@@ -244,9 +247,9 @@ const TradeBox = ({ baseUrl, onStartAgain }: TradeBoxProps): JSX.Element => {
       });
   };
 
-  const cancel = function (): void {
+  const cancel = function (no_confirmation: boolean) {
```

Comment:
If the cancellation needs no confirmation is because it hasn't been taken yet, so we should use `cancel_status` to send the order status we have with the request.